prime-dotnet-webapi/Startup.cs

```diff
@@ -100,6 +100,8 @@ public void ConfigureServices(IServiceCollection services)
                 .AddNewtonsoftJson(options =>
                 {
                     options.SerializerSettings.Converters.Add(new EmptyStringToNullJsonConverter());
+                    // required to fix the error "Self referencing loop detected" caused by PlrViewModel Expertise property
+                    options.SerializerSettings.ReferenceLoopHandling = ReferenceLoopHandling.Ignore;

```

Comment:
Not 100% sure what this is doing, but the other comment might solve this error. But if not we should look into this more since this looks like something we should not be ignoring.

Reply:
Correct, this line is no longer needed. Deleted.